### PR TITLE
Modify application registration method

### PR DESCRIPTION
**Pull request by DmytrMoroz**

### CODE REVIEW

**Review by epam-avramenko — APPROVED** [review]

File: src/org/folio/rest_v2/Eureka.groovy [hunk 1]
```diff
@@ -200,4 +200,51 @@ class Eureka extends Common {
...
+  }
+
+  def registerApplication(String applicationId) {
+    String descriptorsList = getDescriptorsList(applicationId)
+    if (isApplicationRegistered(applicationId)) {
+      logger.warning("Application ${applicationId} is already registered.")
+      return
+    }
+
+    String url = "${Constants.EUREKA_KONG_URL}applications?check=false"
```
Comment (yaroslavishchenko) [inline src/org/folio/rest_v2/Eureka.groovy:238]: This URL can be taken from 'EurekaTenant.kongUrl' attribute

**Review by yaroslavishchenko — APPROVED** [review]

File: src/org/folio/rest_v2/Authorization.groovy [hunk 2]
```diff
@@ -32,6 +32,9 @@ class Authorization extends Common {
     "https://${okapiDomain}${path}"
   }
 
+  String generateKongUrl(String path) {
+    "https://folio-eureka-veselka-kong.ci.folio.org/${path}"
```
[the remark this answers is not in the document]
Comment (DmytrMoroz) [reply src/org/folio/rest_v2/Authorization.groovy]: Fixed

File: src/org/folio/rest_v2/Eureka.groovy [hunk 3]
```diff
@@ -200,4 +200,51 @@ class Eureka extends Common {
       logger.error("Not able to assign ${resourceIds} to ${roleId}")
     }
   }
+
+  boolean isApplicationRegistered(String applicationId) {
+
+    String url = generateKongUrl("/applications/${applicationId}")
```
[the remark this answers is not in the document]
Comment (DmytrMoroz) [reply src/org/folio/rest_v2/Eureka.groovy]: Fixed

File: src/org/folio/rest_v2/Eureka.groovy (hunk 1 above)
Comment (DmytrMoroz) [reply src/org/folio/rest_v2/Eureka.groovy]: Fixed

**Pull request merged**